command/tests/test_identify.py

```diff
@@ -0,0 +1,28 @@
+from unittest import TestCase
+from nose.tools import nottest
+import command.identify as id
+
+class TestGetOperatorArtifactType(TestCase):
+    @nottest
+    def id_test(self, file, expected ):
+        yaml = open(file).read()
+        actual = id.getOperatorArtifactType(yaml)
```

Comment:
`id` is python's built-in, it shouldn't be redefined

Reply:
No longer using the built-in `id` variable